### DISCUSSION TITLE
Hinzufügen von Markern zu Diagrammen

### DISCUSSION THREAD

**Discussion by OE3MNS** [Daveiano/weewx-wdc#216] [Q&A]
Hallo, 

Ich wollte bei Diagrammen div Marker hinzufügen. Lt deiner [Anweisung](https://github.com/Daveiano/weewx-wdc/wiki/Configuration#adding-markers-to-diagrams), doch leider funktioniert das Beispiel nicht. Dass ich mehre, Marker einfügen kann. 
[[[[tempBarometer]]]]
    label = 'Temperature / Barometer'
    [[[[[obs]]]]]
        [[[[[[temp]]]]]]
            markerValue = 0
            markerColor = "#00BFFF"
            observation = "outTemp"
        [[[[[[barometer]]]]]]
            observation = "barometer"
            color = "#5F9EA0"
            markerValue = 1040
            markerColor = "black"

Versuche bei der Luftqualität denn Index in die Grafik einzufügen.[European Air Quality Index](https://airindex.eea.europa.eu/Map/AQI/Viewer/#)

**Meine Konfiguration.**

[[[[aqi]]]]
               # markerValue = 50
               # markerLabel = Schlecht
               # markerColor = "#eb3434"
                label = 'Luftqualität'
                 [[[[[obs]]]]]
                    [[[[[[pm1_0]]]]]]
                        markerValue = 10
                        markerLabel = Sehr Gut
                        markerColor = "#74eb34"
                        observation = "pm1_0"
                        color = "#0058cc"
                    [[[[[[pm2_5]]]]]]
                        markerValue = 20
                        markerLabel = Gut
                        markerColor = "#62eb34"
                        observation = "pm2_5"
                        color = "#3300cc"
                    [[[[[[pm10_0]]]]]]
                        markerValue = 25
                        markerLabel = Mäßig
                        markerColor = "#ebcd34"
                        observation = "pm10_0"
                        color = "#8704d9"

Hab ich da einen Denkfehler ? 

Lg Michi

**Comment by Daveiano** [comment]
Hallo Michi,

wie sieht das Diagram bei dir aus? Habe deine Bsp Konfiguration eben zum Test bei mir eingefügt und es funktioniert:

![GH-216](https://github.com/Daveiano/weewx-wdc/assets/3184897/c31c7dc6-27a3-4e1a-9e11-7e1db3af07c1)

Beide Marker werden angezeigt. Kannst du mir einen Screenshot zeigen wie es bei dir angezeigt wird, bzw. einen Link zu deiner Seite geben?

Das `aqi` Diagram sieht an sich auch gut aus, sehe auf Anhieb nichts falsches. Aber wie gesagt: ohne Screenshot oder Link zur Seite wirds schwer für mich, an sich siehts gut aus.

**Comment by OE3MNS** [comment]
Hallo @Daveiano 

Hier der [Link](https://www.wetter-kreuth.at/) 

Auch beim weewx syslog bekomme ich keine Fehlermeldung, nur in der Grafik wird mir nicht angezeigt. 

Herzlichen Dank
Lg Michi

**Comment by Daveiano** [comment]
Hi Michi,

ich habe mir das ganze genauer angesehen und einen Fehler im Skin gefunden. [Hier](https://github.com/Daveiano/weewx-wdc/files/13210792/main.zip) ist eine aktualisierte Datei, diese bitte herunterladen und entpacken. Dann ins `dist` Verzeichnis innerhalb des `public_html` Verzeichnises kopieren (die Datei `main.js` überschreiben). Ich hoffe danach werden die Marker angezeigt (unter Umständen kann es nötig sein die Seite ohne Cache neuzuladen (STRG+F5)).

Falls das die Lösung ist, wird die Änderung ins nächste Release kommen.

PS: Sehr schöne Seite! Du nutzt die Möglichkeiten des Skins ja voll aus, sehr schön!

**Reply by OE3MNS** [reply]
Herzlichen Dank @Daveiano !!!! 

Jetzt funktioniert es !!! 

Vielen vielen Dank. 

Happy Halloween  🎃
